include/simdjson/portability.h

```diff
@@ -15,6 +15,43 @@
 #define IS_ARM64 1
 #endif
 
+#if defined(__clang__) && defined(_MSC_VER) && defined( IS_X86_64)
+// clang under Visual Studio needs help
+#define __AVX2__ 1
```

Comment:
How will this interact when we're compiling westmere?

Reply:
I don't think it does interact. It is just a macro. The code generation should not be impacted.

However, it might mess with other people importing simdjson and checking the value of the macro. What you'd want is change them, include the x86intrin.h header, then put them back as they were (including undefined). But this sounds like a lot of work.